### PR TITLE
New package: leptonica-1.72

### PR DESCRIPTION
**Pull request by pullmoll**

### CODE REVIEW

File: srcpkgs/leptonica/template [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Template file for 'leptonica'
+pkgname=leptonica
+version=1.72
+revision=1
+build_style=gnu-configure
+makedepends="giflib-devel libpng-devel tiff-devel"
+short_desc="Image processing and analysis library"
+maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
+license="CC-BY-2.5"
```
Comment (beefcurtains) [inline srcpkgs/leptonica/template]: [citation needed]

File: srcpkgs/leptonica/template [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+leptonica-devel_package() {
+	depends="$pkgname>=$version_$revision"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/include/leptonica
+		vmove usr/lib/pkgconfig/lept.pc
+		vmove "usr/lib/*.a"
+		vmove "usr/lib/*.so"
+		vdoc style-guide.txt
+		vdoc version-notes.html
```
Comment (beefcurtains) [inline srcpkgs/leptonica/template]:
```
vdoc moller52.jpg
vlicense leptonica-license.txt
```

File: srcpkgs/leptonica/template (hunk 1 above)
Comment (pullmoll) [inline srcpkgs/leptonica/template]: It's on the [homepage](http://leptonica.org/about-the-license.html) and it's actually CC-BY-3-US. Don't know when that changed...

Comment (beefcurtains) [inline srcpkgs/leptonica/template]: It clearly says on the page they "use the BSD 2-clause license". I think you're confusing the license of their documentation with their source code license.

File: srcpkgs/leptonica/template [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+makedepends="giflib-devel libpng-devel tiff-devel"
+short_desc="Image processing and analysis library"
+maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
+license="CC-BY-3"
+homepage="http://leptonica.org/"
+distfiles="http://leptonica.org/source/$pkgname-$version.tar.gz"
+checksum=79d5eadd32658c9fea38700c975d60aa3d088eaa3e307659f004d40834de1f56
+
+post_install() {
+	vdoc moller52.jpg
```
Comment (beefcurtains) [inline srcpkgs/leptonica/template]: You should put vdoc ``version-notes.html`` in here, that's the doc that references ``moller52.jpg``.

**Comment by xtraeme** [comment]
leptonic is `2-clause-BSD` licensed, see http://leptonica.org/about-the-license.html

**Comment by pullmoll** [comment]
Rebased it all in one

**Pull request merged**